### ISSUE TITLE
Remote State Backend S3 Bucket missing attributes in GovCloud

### ISSUE THREAD

**Issue by ryno75** [gruntwork-io/terragrunt#1369]
When creating the Terraform Remote State S3 Bucket with the standard Terragrunt `remote_state` and `config` blocks under the Gov Cloud region (us-gov-west-1), the `encrypt` and `s3_bucket_tags` attributes are not applied to the resultant backend S3 bucket when Terragrunt creates it.  In addition, S3 bucket versioning is not enabled even though the `skip_bucket_versioning` attribute is not set (default is `false`).

## Example config:
```hcl
remote_state {
  backend = "s3"

  config = {
    region         = "us-gov-west-1"
    encrypt        = true
    bucket         = "org.foo.terraform-state"
    key            = "${path_relative_to_include()}/terraform.tfstate"
    dynamodb_table = "org.foo.terraform-state"

    s3_bucket_tags = {
      env     = "dev"
      product = "foo"
    }

    dynamodb_table_tags = {
      env     = "dev"
      product = "foo"
    }
  }
}
```

## Expected Behavior:
By supplying the standard encryption and tag attributes in the `remote_state` `config` block, the resultant resources created by Terragrunt would have those tags and encryption settings applied.
```sh
❯ aws s3api get-bucket-tagging --bucket gov.snafu.foo.dev.terraform
{
    "TagSet": [
        {
            "Key": "env",
            "Value": "dev"
        },
        {
            "Key": "product",
            "Value": "foo"
        }
    ]
}

❯ aws s3api get-bucket-encryption --bucket gov.snafu.foo.dev.terraform
{
    "ServerSideEncryptionConfiguration": {
        "Rules": [
            {
                "ApplyServerSideEncryptionByDefault": {
                    "SSEAlgorithm": "AES256"
                }
            }
        ]
    }
}

❯ aws s3api get-bucket-versioning --bucket gov.snafu.foo.dev.terraform
{
    "Status": "Enabled"
}
```

## Actual Behavior:
While the DynamoDB table _is_ getting its tags created; the S3 bucket is neither getting its tags or its encryption set...
```
❯ terragrunt plan
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:44 Running command: terraform --version
[terragrunt] 2020/09/23 23:59:45 Terraform version: 0.13.3
[terragrunt] 2020/09/23 23:59:45 Reading Terragrunt config file at /homedir/code/terragrunt_proj/live/dev/iam/terragrunt.hcl
[terragrunt] 2020/09/23 23:59:52 Copying files from /homedir/code/terragrunt_proj/live/dev/iam into /homedir/code/terragrunt_proj/live/dev/iam/.terragrunt-cache/NmUb0NQowR_kidRL6AqXOQfqPAw/jEuKqkF7Y7XrbxJ9C_9A3LTh-5I
[terragrunt] 2020/09/23 23:59:52 Setting working directory to /homedir/code/terragrunt_proj/live/dev/iam/.terragrunt-cache/NmUb0NQowR_kidRL6AqXOQfqPAw/jEuKqkF7Y7XrbxJ9C_9A3LTh-5I
[terragrunt] 2020/09/23 23:59:52 Generated file /homedir/code/terragrunt_proj/live/dev/iam/.terragrunt-cache/NmUb0NQowR_kidRL6AqXOQfqPAw/jEuKqkF7Y7XrbxJ9C_9A3LTh-5I/providers.tf.
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:52 Initializing remote state for the s3 backend
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:52 [terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam]  Remote state S3 bucket gov.snafu.foo.dev.terraform does not exist or you don't have permissions to access it. Would you like Terragrunt to create it? (y/n)
y
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:54 Create S3 bucket gov.snafu.foo.dev.terraform
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:54 Creating S3 bucket gov.snafu.foo.dev.terraform
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:55 Waiting for bucket gov.snafu.foo.dev.terraform to be created
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:55 S3 bucket gov.snafu.foo.dev.terraform created.
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:55 Enabling root access to S3 bucket gov.snafu.foo.dev.terraform
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:55 Create S3 bucket gov.snafu.foo.dev.terraform returned an error: InvalidClientTokenId: The security token included in the request is invalid
	status code: 403, request id: a86716b9-42ac-49ae-a70e-ac7b598befdf. Sleeping for 10s and will try again.
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:05 Create S3 bucket gov.snafu.foo.dev.terraform
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:05 Creating S3 bucket gov.snafu.foo.dev.terraform
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:05 Looks like someone is creating bucket gov.snafu.foo.dev.terraform at the same time. Will not attempt to create it again.
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:05 Waiting for bucket gov.snafu.foo.dev.terraform to be created
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:05 S3 bucket gov.snafu.foo.dev.terraform created.
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:05 WARNING: Versioning is not enabled for the remote state S3 bucket gov.snafu.foo.dev.terraform. We recommend enabling versioning so that you can roll back to previous versions of your Terraform state in case of error.
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:05 Lock table gov.snafu.foo.dev.terraform-lock-table does not exist in DynamoDB. Will need to create it just this first time.
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:05 Creating table gov.snafu.foo.dev.terraform-lock-table in DynamoDB
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:06 Table gov.snafu.foo.dev.terraform-lock-table is not yet in active state. Will check again after 10s.
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:16 Success! Table gov.snafu.foo.dev.terraform-lock-table is now in active state.
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:16 Adding tags to lock table: map[env:dev product:foo]
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/24 00:00:16 Running command: terraform init -backend-config=bucket=gov.snafu.foo.dev.terraform -backend-config=dynamodb_table=gov.snafu.foo.dev.terraform-lock-table -backend-config=encrypt=true -backend-config=key=iam/terraform.tfstate -backend-config=region=us-gov-west-1 -lock-timeout=20m
Initializing modules...

❯ aws s3api get-bucket-tagging --bucket gov.snafu.foo.dev.terraform
An error occurred (NoSuchTagSet) when calling the GetBucketTagging operation: The TagSet does not exist

❯ aws s3api get-bucket-encryption --bucket gov.snafu.foo.dev.terraform
An error occurred (ServerSideEncryptionConfigurationNotFoundError) when calling the GetBucketEncryption operation: The server side encryption configuration was not found

❯ aws s3api get-bucket-versioning --bucket gov.snafu.foo.dev.terraform

```

**Comment by brikis98** [comment]
We unfortunately don't run tests against AWS GovCloud right now, and it does require some changes to how you make API calls, so such a bug is possible 😕 

This is particularly suspicious:

```
[terragrunt] [/homedir/code/terragrunt_proj/live/dev/iam] 2020/09/23 23:59:55 Create S3 bucket gov.snafu.foo.dev.terraform returned an error: InvalidClientTokenId: The security token included in the request is invalid
	status code: 403, request id: a86716b9-42ac-49ae-a70e-ac7b598befdf. Sleeping for 10s and will try again.
```

I'm wondering if the code that updates the S3 bucket (a) is doing something wrong so it doesn't work with GovClod and (b) is not handling the errors from that correctly, so it running along as if everything is OK. If anyone has a GovCloud account and some time to poke around and see what's happening, a PR with a fix is very much appreciated 👍

**Comment by ryno75** [comment]
That was my suspicion as well.  I have planned on trying to carve out a little time to resolve this and submit a PR.  Not sure how soon I can get to it but... Definitely something my company will want/need fixed so I have no roadblock working on it during normal business hours (other than prioritization).

**Comment by ryno75** [comment]
I believe this was resolved by https://github.com/gruntwork-io/terragrunt/releases/tag/v0.29.1
There is another GovCloud related issue I am going to open.

**Comment by ryno75** [comment]
Reopening as the originally reported problems still exist, only the error/root-cause has changed.

**Comment by ryno75** [comment]
Now that the credential error has been resolved by #1654 the an additional error is resulting in the same issue reported in the opening comment.  

Here is the current trace run output:
```
❯ terragrunt init --terragrunt-log-level trace
DEBU[0000] Found locals block: evaluating the expressions.
DEBU[0000] Evaluated 1 locals (remaining 1): backend_name
DEBU[0000] Evaluated 1 locals (remaining 0): backend_tags
DEBU[0000] Running command: terraform --version          prefix=[/tmp/tgtest/live/test]
DEBU[0001] Terraform version: 0.15.5
DEBU[0001] Reading Terragrunt config file at /tmp/tgtest/live/test/terragrunt.hcl
DEBU[0001] Found locals block: evaluating the expressions.
DEBU[0001] Evaluated 1 locals (remaining 1): backend_name
DEBU[0001] Evaluated 1 locals (remaining 0): backend_tags
DEBU[0001] Downloading Terraform configurations from file:///tmp/tgtest/modules into /tmp/tgtest/live/test/.terragrunt-cache/PKlWVAzIyzq6v6fdHQxwvld3i4U/3ntBAGK9C6eOQO6Mp9y0HxBwpmk
DEBU[0001] Copying files from /tmp/tgtest/live/test into /tmp/tgtest/live/test/.terragrunt-cache/PKlWVAzIyzq6v6fdHQxwvld3i4U/3ntBAGK9C6eOQO6Mp9y0HxBwpmk/test
DEBU[0001] Setting working directory to /tmp/tgtest/live/test/.terragrunt-cache/PKlWVAzIyzq6v6fdHQxwvld3i4U/3ntBAGK9C6eOQO6Mp9y0HxBwpmk/test
DEBU[0001] Generated file /tmp/tgtest/live/test/.terragrunt-cache/PKlWVAzIyzq6v6fdHQxwvld3i4U/3ntBAGK9C6eOQO6Mp9y0HxBwpmk/test/backend.tf.  prefix=[/tmp/tgtest/live/test]
DEBU[0001] Initializing remote state for the s3 backend  prefix=[/tmp/tgtest/live/test]
Remote state S3 bucket tg-govcloud-backend-test0 does not exist or you don't have permissions to access it. Would you like Terragrunt to create it? (y/n) y
DEBU[0003] Create S3 bucket with retry tg-govcloud-backend-test0  prefix=[/tmp/tgtest/live/test]
DEBU[0003] Creating S3 bucket tg-govcloud-backend-test0  prefix=[/tmp/tgtest/live/test]
DEBU[0003] Created S3 bucket tg-govcloud-backend-test0   prefix=[/tmp/tgtest/live/test]
DEBU[0003] Waiting for bucket tg-govcloud-backend-test0 to be created  prefix=[/tmp/tgtest/live/test]
DEBU[0003] S3 bucket tg-govcloud-backend-test0 created.  prefix=[/tmp/tgtest/live/test]
DEBU[0003] Enabling root access to S3 bucket tg-govcloud-backend-test0  prefix=[/tmp/tgtest/live/test]
ERRO[0004] Create S3 bucket with retry tg-govcloud-backend-test0 returned an error: MalformedPolicy: Invalid principal in policy
	status code: 400, request id: 35RREQYK14ZXHNW7, host id: 11jdEp/NYdIf7PAHOoSgf4JhqYMZ2Opm9h5TJPNHrMg0SrbyQ2WN1UZCJOH1YRQm6t0XPMDZC2o=. Sleeping for 10s and will try again.  prefix=[/tmp/tgtest/live/test]
DEBU[0014] Create S3 bucket with retry tg-govcloud-backend-test0  prefix=[/tmp/tgtest/live/test]
DEBU[0014] Creating S3 bucket tg-govcloud-backend-test0  prefix=[/tmp/tgtest/live/test]
DEBU[0014] Looks like you're already creating bucket tg-govcloud-backend-test0 at the same time. Will not attempt to create it again.  prefix=[/tmp/tgtest/live/test]
DEBU[0014] Waiting for bucket tg-govcloud-backend-test0 to be created  prefix=[/tmp/tgtest/live/test]
DEBU[0014] S3 bucket tg-govcloud-backend-test0 created.  prefix=[/tmp/tgtest/live/test]
WARN[0014] Versioning is not enabled for the remote state S3 bucket tg-govcloud-backend-test0. We recommend enabling versioning so that you can roll back to previous versions of your Terraform state in case of error.  prefix=[/tmp/tgtest/live/test]
DEBU[0014] Lock table tg-govcloud-backend-test0 does not exist in DynamoDB. Will need to create it just this first time.  prefix=[/tmp/tgtest/live/test]
DEBU[0014] Creating table tg-govcloud-backend-test0 in DynamoDB  prefix=[/tmp/tgtest/live/test]
DEBU[0014] Table tg-govcloud-backend-test0 is not yet in active state. Will check again after 10s.  prefix=[/tmp/tgtest/live/test]
DEBU[0025] Success! Table tg-govcloud-backend-test0 is now in active state.  prefix=[/tmp/tgtest/live/test]
DEBU[0025] Adding tags to lock table: map[Env:test Name:tg-govcloud-backend-test0 Owner:ryno75]  prefix=[/tmp/tgtest/live/test]
DEBU[0025] Running command: terraform init               prefix=[/tmp/tgtest/live/test]
```

All expected outputs fail as previously mentioned...
```
❯ aws s3api get-bucket-tagging --bucket tg-govcloud-backend-test0
An error occurred (NoSuchTagSet) when calling the GetBucketTagging operation: The TagSet does not exist

❯ aws s3api get-bucket-encryption --bucket tg-govcloud-backend-test0
An error occurred (ServerSideEncryptionConfigurationNotFoundError) when calling the GetBucketEncryption operation: The server side encryption configuration was not found

❯ aws s3api get-bucket-versioning --bucket tg-govcloud-backend-test0
```

My hunch here was that the ARN "partition" was set to the commercial value of `aws` and NOT the required GovCloud parition value of "aws-us-gov".  An inspection of the API event in Cloudtrail quickly confirmed this suspicion...
```
{
    "eventVersion": "1.08",
    ...<REDACTED>...
    "eventSource": "s3.amazonaws.com",
    "eventName": "PutBucketPolicy",
    "awsRegion": "us-gov-west-1",
    "userAgent": "[aws-sdk-go/1.37.19 (go1.16.2; darwin; amd64)]",
    "errorCode": "MalformedPolicy",
    "errorMessage": "Invalid principal in policy",
    "requestParameters": {
        "bucketPolicy": {
            "Version": "2012-10-17",
            "Statement": [
                {
                    "Action": "s3:*",
                    "Effect": "Allow",
                    "Principal": {
                        "AWS": [
                            "arn:aws:iam::<REDACTED>:root"
                        ]
                    },
                    "Resource": [
                        "arn:aws:s3:::tg-govcloud-backend-test0",
                        "arn:aws:s3:::tg-govcloud-backend-test0/*"
                    ],
                    "Sid": "RootAccess"
                }
            ]
        },
        "bucketName": "tg-govcloud-backend-test0",
        "Host": "tg-govcloud-backend-test0.s3.us-gov-west-1.amazonaws.com",
        "policy": ""
    },
    "responseElements": null,
    ...<REDACTED>...
    "eventType": "AwsApiCall",
}
```
Notice the ARN in the statement policy: `arn:aws:iam::<REDACTED>:root` contains the standard AWS partition (**`aws`**) and **_NOT_** the AWS GovCloud partition (**`aws-us-gov`**)

**Comment by ryno75** [comment]
Tested with build from #1718 with success...
```
❯ ~/bin/terragrunt init --terragrunt-log-level trace
DEBU[0000] Found locals block: evaluating the expressions.
DEBU[0000] Evaluated 2 locals (remaining 1): region, state_name
DEBU[0000] Evaluated 1 locals (remaining 0): state_tags
DEBU[0000] Running command: terraform --version          prefix=[/tmp/tgtest/live]
DEBU[0001] Terraform version: 0.15.5
DEBU[0001] Reading Terragrunt config file at /tmp/tgtest/live/terragrunt.hcl
DEBU[0001] Found locals block: evaluating the expressions.
DEBU[0001] Evaluated 2 locals (remaining 1): region, state_name
DEBU[0001] Evaluated 1 locals (remaining 0): state_tags
DEBU[0001] Downloading Terraform configurations from file:///tmp/tgtest/modules into /tmp/tgtest/live/.terragrunt-cache/Hyz2YYCH4QEWMXH_nbrK3K2OQMg/o70VZL4sauClAGMX_iRhS5PktNg
DEBU[0001] Copying files from /tmp/tgtest/live into /tmp/tgtest/live/.terragrunt-cache/Hyz2YYCH4QEWMXH_nbrK3K2OQMg/o70VZL4sauClAGMX_iRhS5PktNg/test
DEBU[0001] Setting working directory to /tmp/tgtest/live/.terragrunt-cache/Hyz2YYCH4QEWMXH_nbrK3K2OQMg/o70VZL4sauClAGMX_iRhS5PktNg/test
DEBU[0001] Generated file /tmp/tgtest/live/.terragrunt-cache/Hyz2YYCH4QEWMXH_nbrK3K2OQMg/o70VZL4sauClAGMX_iRhS5PktNg/test/provider.tf.  prefix=[/tmp/tgtest/live]
DEBU[0001] Generated file /tmp/tgtest/live/.terragrunt-cache/Hyz2YYCH4QEWMXH_nbrK3K2OQMg/o70VZL4sauClAGMX_iRhS5PktNg/test/backend.tf.  prefix=[/tmp/tgtest/live]
DEBU[0001] Initializing remote state for the s3 backend  prefix=[/tmp/tgtest/live]
Remote state S3 bucket tg-govcloud-backend-test-issue1369 does not exist or you don't have permissions to access it. Would you like Terragrunt to create it? (y/n) y
DEBU[0006] Create S3 bucket with retry tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0006] Creating S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0006] Created S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0006] Waiting for bucket tg-govcloud-backend-test-issue1369 to be created  prefix=[/tmp/tgtest/live]
DEBU[0007] S3 bucket tg-govcloud-backend-test-issue1369 created.  prefix=[/tmp/tgtest/live]
DEBU[0007] Enabling root access to S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0007] Enabled root access to bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0007] Enabling enforced TLS access for S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0007] Enabled enforced TLS access for bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0007] Blocking all public access to S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0007] Blocked all public access to S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0007] Tagging S3 bucket with map[Env:test Name:tg-govcloud-backend-test-issue1369 Owner:ryno75]  prefix=[/tmp/tgtest/live]
DEBU[0008] Tagged S3 bucket with map[Env:test Name:tg-govcloud-backend-test-issue1369 Owner:ryno75]  prefix=[/tmp/tgtest/live]
DEBU[0008] Enabling versioning on S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0008] Enabled versioning on S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0008] Enabling bucket-wide SSE on AWS S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0008] Enabled bucket-wide SSE on AWS S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0008] Access Logging is disabled for the remote state AWS S3 bucket tg-govcloud-backend-test-issue1369  prefix=[/tmp/tgtest/live]
DEBU[0008] Lock table tg-govcloud-backend-test-issue1369 does not exist in DynamoDB. Will need to create it just this first time.  prefix=[/tmp/tgtest/live]
DEBU[0008] Creating table tg-govcloud-backend-test-issue1369 in DynamoDB  prefix=[/tmp/tgtest/live]
DEBU[0008] Table tg-govcloud-backend-test-issue1369 is not yet in active state. Will check again after 10s.  prefix=[/tmp/tgtest/live]
DEBU[0018] Success! Table tg-govcloud-backend-test-issue1369 is now in active state.  prefix=[/tmp/tgtest/live]
DEBU[0018] Adding tags to lock table: map[Env:test Name:tg-govcloud-backend-test-issue1369 Owner:ryno75]  prefix=[/tmp/tgtest/live]
DEBU[0018] Running command: terraform init               prefix=[/tmp/tgtest/live]

Initializing the backend...

Successfully configured the backend "s3"! Terraform will automatically
use this backend unless the backend configuration changes.

Initializing provider plugins...
- Finding latest version of hashicorp/aws...
- Installing hashicorp/aws v3.45.0...
- Installed hashicorp/aws v3.45.0 (signed by HashiCorp)

Terraform has created a lock file .terraform.lock.hcl to record the provider
selections it made above. Include this file in your version control repository
so that Terraform can guarantee to make the same selections by default when
you run "terraform init" in the future.

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
DEBU[0030] Copying lock file from /tmp/tgtest/live/.terragrunt-cache/Hyz2YYCH4QEWMXH_nbrK3K2OQMg/o70VZL4sauClAGMX_iRhS5PktNg/test/.terraform.lock.hcl to /tmp/tgtest/live  prefix=[/tmp/tgtest/live]
```

```
❯ aws s3api get-bucket-tagging --bucket tg-govcloud-backend-test-issue1369
{
    "TagSet": [
        {
            "Key": "Env",
            "Value": "test"
        },
        {
            "Key": "Owner",
            "Value": "ryno75"
        },
        {
            "Key": "Name",
            "Value": "tg-govcloud-backend-test-issue1369"
        }
    ]
}

❯ aws s3api get-bucket-versioning --bucket tg-govcloud-backend-test-issue1369
{
    "Status": "Enabled"
}

❯ aws s3api get-bucket-encryption --bucket tg-govcloud-backend-test-issue1369
{
    "ServerSideEncryptionConfiguration": {
        "Rules": [
            {
                "ApplyServerSideEncryptionByDefault": {
                    "SSEAlgorithm": "aws:kms"
                },
                "BucketKeyEnabled": false
            }
        ]
    }
}
```
